ffi/capi/cpp/examples/fixeddecimal_wasm/crate/LICENSE

```diff
@@ -0,0 +1,44 @@
+UNICODE LICENSE V3
```

Comment:
Question/Issue, here and elsewhere: do we need to copy this LICENSE file here? The crate isn't being published

Reply:
the license file CI doesn't know about `publish = false`. I believe we generally add the license file instead of manually excluding crates

Reply:
OK as long as this is covered by `cargo tidy` to make sure we don't have version skew of LICENSE files